tools/ci/download-xs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+
+# Download HDF5 data
+if [[ ! -e $HOME/nndc_hdf5/cross_sections.xml ]]; then
+    wget -q -O - https://anl.box.com/shared/static/teaup95cqv8s9nn56hfn7ku8mmelr95p.xz | tar -C $HOME -xJ
+fi
+
+# Download ENDF/B-VII.1 distribution
+ENDF=$HOME/endf-b-vii.1/
+if [[ ! -d $ENDF/neutrons || ! -d $ENDF/photoat || ! -d $ENDF/atomic_relax ]]; then
+    wget -q -O - https://anl.box.com/shared/static/4kd2gxnf4gtk4w1c8eua5fsua22kvgjb.xz | tar -C $HOME -xJ
```

Comment:
If there's a scenario where part of this package exists, it might make sense to delete the entire package and extract again:
```
rm -rf $ENDF > /dev/null 2>&1
```